types/hapi/test/route/additional-options.ts

```diff
@@ -6,6 +6,29 @@ var authConfig: Hapi.RouteAdditionalConfigurationOptions = {
   app: {}
 };
 
+var extConfigSingle: Hapi.RouteAdditionalConfigurationOptions = {
+  ext: {
+    type: 'onPreAuth',
+    method: <Hapi.ServerExtRequestHandler> function (request, reply) {
+      reply('ok');
+    }
+  }
+}
+
+var extConfigMulti: Hapi.RouteAdditionalConfigurationOptions = {
+  ext: [{
+    type: 'onPreAuth',
+    method: <Hapi.ServerExtRequestHandler> function (request, reply) {
+      reply('ok');
+    }
+  }, {
+    type: 'onPostAuth',
+    method: <Hapi.ServerExtRequestHandler> function (request, reply) {
+      reply('ok');
+    }
+  }]
```

Comment:
That was my probably my fault as the indentation of the lines above are incorrect:
```
var authConfig: Hapi.RouteAdditionalConfigurationOptions = {
  app: {},  // only 2 spaces
};
```